Application/EdFi.Ods.AdminApi/Infrastructure/WebApplicationBuilderExtensions.cs

```diff
@@ -21,6 +21,8 @@
 using EdFi.Ods.AdminApi.Helpers;
 using EdFi.Ods.AdminApi.Infrastructure.Context;
 using EdFi.Common.Extensions;
+using System.Configuration;
+using Microsoft.AspNetCore.Mvc.Filters;

```

Comment:
If you press Ctrl-R Ctrl-G you can remove these `usings` along with others that are not necessary on this file.